applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.h

```diff
@@ -399,7 +401,87 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ConstitutiveLawUtilities
         const MatrixType& rRe
         );
 
-  private:
+    /**
+     * @brief This computes the mapper operator between the stresses in the isotropic
+     * "ficticious" space and the real anisotropic space
+     * @param rValues The values of the constitutive la
+     * @param rAs The mapper operator
+     * @param rAs The mapper operator inverse
+     * @note Eq.(2.39) S. Oller book: Comportamiento mecánico de los materiales compuestos
+     */
+    static void CalculateAnisotropicStressMapperMatrix(
+        ConstitutiveLaw::Parameters &rValues,
+        Matrix &rAs,
```

Comment:
Consistency, put the & in the same side (I prefer left)

Reply:
Parameters object is a slow object. should not used inside CL